.github/workflows/release.yml

```
@@ -39,6 +39,16 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - uses: elastic/apm-pipeline-library/.github/actions/github-token@main
```

`current` points to the latest stable version

```suggestion
      - uses: elastic/apm-pipeline-library/.github/actions/github-token@current
```